bounties/other/openfire-fastpath-plugin/1/README.md

```diff
@@ -0,0 +1,17 @@
+# Description
+
+`openfire-fastpath-plugin` is vulnerable to `Cross-Site Scripting (XSS)`.
+
+# Steps To Reproduce
+
+1. Download openfire and install https://www.igniterealtime.org/downloads/
+2. Run the server http://localhost:9090/index.jsp
+3. Click on "Plugins" http://localhost:9090/plugin-admin.jsp and install the "fastpath" plugin.
+4. Click on  "fastpath" and "create workgroup" http://localhost:9090/plugins/fastpath/workgroup-create.jsp
+```
```

Comment:
@ready-research Please remove these tilts. Currently, the readme looks like this:

![image](https://user-images.githubusercontent.com/16623935/102345791-ebaba780-3fc3-11eb-91fb-380f6b7d84f2.png)

Reply:
updated